tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_NA_3km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta.yaml

```diff
@@ -37,3 +37,7 @@ task_run_fcst:
 task_run_post:
   NNODES_RUN_POST: 8
   PPN_RUN_POST: 12
+task_run_prdgen:
+  RUN_TASK_RUN_PRDGEN: true
```

Comment:
This should be in `workflow_sections`.

Reply:
Yes it looks like it should fall under workflow_switches.  I'll fix that along with your other suggested changes below and push the fixes to GitHub soon - thanks for reviewing!